main.go

```diff
@@ -0,0 +1,203 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"io"
+	_ "net/http/pprof"
+	"net/url"
+	"os"
+	"os/signal"
+	"runtime"
+	"time"
+
+	"github.com/go-kit/kit/log"
+	"github.com/goph/emperror"
+	"github.com/gorilla/mux"
+	"github.com/spf13/pflag"
+	"github.com/spf13/viper"
+
+	"github.com/xmidt-org/webpa-common/concurrent"
+	"github.com/xmidt-org/webpa-common/logging"
+	"github.com/xmidt-org/webpa-common/server"
+	"github.com/xmidt-org/webpa-common/webhook"
+	"github.com/xmidt-org/webpa-common/webhook/aws"
+	"github.com/xmidt-org/webpa-common/xwebhook"
+)
+
+const (
+	applicationName, apiBase = "hecate", "/api/v1"
+	DEFAULT_KEY_ID           = "current"
+)
+
+var (
+	GitCommit = "undefined"
+	Version   = "undefined"
+	BuildTime = "undefined"
+)
+
+type Config struct {
+	Webhook xwebhook.Config
+}
+
+func hecate(arguments []string) int {
```

Comment:
We don't want to use uber fx for this application? I thought the goal was to use it for any new applications moving forward?

Reply:
We should if we can! I just didn't make time to explore how nicely the old webhook library would play with fx. @j-mai has some free cycles now and she might give it a try! 🚀